W-11918979-EOL-xref-error-v1.1-dm (#199)
mulesoft/docs-dataweave: modules/ROOT/pages/dataweave-reference-documentation.adoc

When defining an input of type CSV, there are a few optional parameters you can  add to the input directive to customize how the data is parsed. These are not defined in the DataWeave script but on the Mule XML code, in the Transform Message XML element.

> In Anypoint Studio there are two ways to achieve this. You can either manually add the attributes to the project's XML, or do it through the graphical interface, by selecting the element from the tree view in the input section and clicking the gear icon.

Reviewer: ```suggestion
In Anypoint Studio, there are two ways to achieve this. You can either manually add the attributes to the project's XML or use the graphical interface by selecting the element from the tree view in the input section and clicking the gear icon.
```

==== Global MEL Functions

> Your DataWeave code can call any function you define as a global xref:3.8@mule-runtime::mule-expression-language-mel.adoc[Mule Expression Language (MEL)] function, as long as it is correctly defined in the Mule Project where your Transform Message element sits.

Reviewer: ```suggestion
Your DataWeave code can call any function that you define as a global xref:3.8@mule-runtime::mule-expression-language-mel.adoc[Mule Expression Language (MEL)] function, as long as it is correctly defined in the Mule project where your Transform Message element sits.
```
